rest/views.py

```diff
@@ -48,6 +49,39 @@
 from django.http.response import Http404
 
 
+def get_urls(raw_urls, list_urls, urlbase=''):
+    """
+    Recursively builds a list of all the urls in the current project and
+    the name of their associated view
+    """
+    list_urls = list_urls or []
+    from operator import itemgetter
+    for entry in raw_urls:
+        full_url = (urlbase + entry.regex.pattern).replace('^', '')
+        if '$' in full_url:
+            full_url = full_url[0: full_url.index("$")]
+        if entry.callback:  # if it points to a view
+            list_urls.append({"pattern": full_url})
+        else:  # if it points to another urlconf, recur!
+            get_urls(entry.url_patterns, list_urls, full_url)
+
+    # sort alphabetically
+    list_urls = sorted(list_urls, key=itemgetter('pattern'))
```

Comment:
nit: why not sorting in place? Allocating a new variable here is not that useful